src/styles/card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+  transition-property: background-color, color, border;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    font-family: $font;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-color;
+    padding-bottom: 4px;
+  }
+
+  &__productCode {
+    font-family: $font;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: $fontWeightStandard;
+    padding-bottom: 16px;
+    color: $secondColor;
+  }
+
+  &__buyButton {
```

Comment:
The `&__buyButton` could benefit from the use of the `font` shorthand property as well. For example: `font: 700 14px/16px inherit;`. Note that `inherit` can be used for the font-family if it's the same as the parent element.